configure

```diff
@@ -977,9 +977,19 @@ def configure_openssl(o):
 
 
   if options.without_ssl:
+    if options.shared_openssl:
+      withoutSSLError('--shared-openssl')
+    if options.openssl_no_asm:
+      withoutSSLError('--openssl-no-asm')
+    if options.openssl_fips:
+      withoutSSLError('--openssl-fips')
     return
   configure_library('openssl', o)
 
+def withoutSSLError(option):
+  print('Error: --without-ssl is incompatible with %s' % option)
+  exit(1);
```

Comment:
Style: `without_ssl_error` and no semicolon.

You could define it in the `if options.without_ssl:` block if that's the only place you use it (or simply use `or` to chain the conditions.)